From: Matthias Geerdsen <matthias@vorlons.info>
Date: Sat, 25 Apr 2015 15:33:36 +0200
Subject: remove reference to gcrypt.h related on update to gnutls28-dev

---
 src/net/tls/TLSSession.cpp | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/src/net/tls/TLSSession.cpp b/src/net/tls/TLSSession.cpp
index 68f4266..18fc622 100644
--- a/src/net/tls/TLSSession.cpp
+++ b/src/net/tls/TLSSession.cpp
@@ -27,7 +27,6 @@
 
 #if VMIME_HAVE_PTHREAD
 #	include <pthread.h>
-#	include <gcrypt.h>
 #	include <errno.h>
 #endif // VMIME_HAVE_PTHREAD
 
@@ -45,14 +44,6 @@
 #endif // VMIME_DEBUG && GNUTLS_DEBUG
 
 
-#if VMIME_HAVE_PTHREAD && defined(GCRY_THREAD_OPTION_PTHREAD_IMPL)
-extern "C"
-{
-	GCRY_THREAD_OPTION_PTHREAD_IMPL;
-}
-#endif // VMIME_HAVE_PTHREAD && defined(GCRY_THREAD_OPTION_PTHREAD_IMPL
-
-
 namespace vmime {
 namespace net {
 namespace tls {
@@ -65,10 +56,6 @@ struct TLSGlobal
 {
 	TLSGlobal()
 	{
-#if VMIME_HAVE_PTHREAD && defined(GCRY_THREAD_OPTION_PTHREAD_IMPL)
-		gcry_control(GCRYCTL_SET_THREAD_CBS, &gcry_threads_pthread);
-#endif // VMIME_HAVE_PTHREAD && defined(GCRY_THREAD_OPTION_PTHREAD_IMPL
-
 		gnutls_global_init();
 		//gnutls_global_init_extra();
 
