From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sat, 25 Apr 2015 13:20:50 +0200
Subject: changing compiler flags for nonansi

---
 SConstruct | 14 ++++++--------
 1 file changed, 6 insertions(+), 8 deletions(-)

diff --git a/SConstruct b/SConstruct
index e63a7de..688684a 100644
--- a/SConstruct
+++ b/SConstruct
@@ -595,8 +595,6 @@ env.Append(CPPDEFINES = ['_REENTRANT=1'])
 
 env.Append(CXXFLAGS = ['-W'])
 env.Append(CXXFLAGS = ['-Wall'])
-env.Append(CXXFLAGS = ['-ansi'])
-env.Append(CXXFLAGS = ['-pedantic'])
 env.Append(CXXFLAGS = ['-Wpointer-arith'])
 env.Append(CXXFLAGS = ['-Wold-style-cast'])
 env.Append(CXXFLAGS = ['-Wconversion'])
@@ -609,8 +607,6 @@ env.Append(TARFLAGS = ['--bzip2'])
 if env['debug'] == 'yes':
 	env.Append(CXXFLAGS = ['-g'])
 	env.Append(CXXFLAGS = ['-O0'])
-else:
-	env.Append(CXXFLAGS = ['-O2'])
 	#-fomit-frame-pointer -fmove-all-movables -fstrict-aliasing
 
 #env.Append(LIBS = ['additional-lib-here'])
@@ -1299,7 +1295,9 @@ VMIME_ADDITIONAL_PC_LIBS=""
 # Check compilers, processors, etc
 #
 
-AC_PROG_CC
+AC_PROG_CC_STDC
+AC_USE_SYSTEM_EXTENSIONS
+AC_SYS_LARGEFILE
 AC_PROG_CXX
 AC_PROG_CPP
 AC_C_CONST
@@ -1802,8 +1800,8 @@ AC_SUBST(PKGCONFIG_LIBS)
 EXTRA_CFLAGS="$EXTRA_CFLAGS -D_REENTRANT=1 -D_THREAD_SAFE=1 $LIBGNUTLS_CFLAGS"
 EXTRA_LIBS="$GSASL_LIBS $LIBGNUTLS_LIBS"
 
-CFLAGS=""
-CXXFLAGS=""
+CFLAGS+=""
+CXXFLAGS+=""
 
 # -- Debug
 if test x$VMIME_DEBUG = x1 ; then
@@ -1831,7 +1829,7 @@ EXTRA_CFLAGS="$EXTRA_CFLAGS $lt_prog_compiler_pic"
 
 """)
 
-	compilerFlags = [ '-ansi', '-pedantic', '-W', '-Wall', '-Wpointer-arith', '-Wold-style-cast', '-Wconversion' ]
+	compilerFlags = [ '-W', '-Wall', '-Wpointer-arith', '-Wold-style-cast', '-Wconversion' ]
 
 	for f in compilerFlags:
 		configure_in.write('# ' + f + '\n')
