From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sat, 25 Apr 2015 13:18:10 +0200
Subject: check if a string contains qp in buffers

---
 src/wordEncoder.cpp | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/wordEncoder.cpp b/src/wordEncoder.cpp
index 67bd7a1..b62b9ea 100644
--- a/src/wordEncoder.cpp
+++ b/src/wordEncoder.cpp
@@ -239,6 +239,13 @@ bool wordEncoder::isEncodingNeeded(const string& buffer, const charset& charset)
 	if (buffer.find_first_of("\n\r") != string::npos)
 		return true;
 
+	// If the string contains a QP string, we need to encode this.
+	// Not a 100% check, but we'd only get more encoded strings.
+	std::string::size_type pos = buffer.find("=?");
+	std::string::size_type end = buffer.find("?=");
+	if (pos != string::npos && end != string::npos && end > pos)
+		return true;
+
 	return false;
 }
 
