From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sat, 25 Apr 2015 13:35:02 +0200
Subject: ignore multiple colons in headers

---
 src/headerField.cpp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/headerField.cpp b/src/headerField.cpp
index fce9332..a8fc973 100644
--- a/src/headerField.cpp
+++ b/src/headerField.cpp
@@ -132,7 +132,9 @@ ref <headerField> headerField::parseNext(const string& buffer, const string::siz
 				                  buffer.begin() + nameEnd);
 
 				// Skip ':' character
-				++pos;
+				// Patch: don't just skip one colon, but any and all, if available. NS 2 December 2013
+				while ((pos < end) && (buffer[pos] == ':'))
+				    ++pos;
 
 				// Skip spaces between ':' and the field contents
 				while (pos < end && (buffer[pos] == ' ' || buffer[pos] == '\t'))
