From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sat, 25 Apr 2015 13:05:28 +0200
Subject: try to catch the correct charset

---
 src/word.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/word.cpp b/src/word.cpp
index db720dc..6526042 100644
--- a/src/word.cpp
+++ b/src/word.cpp
@@ -687,7 +687,13 @@ const string word::getConvertedText(const charset& dest) const
 {
 	string out;
 
+	try {
 	charset::convert(m_buffer, out, m_charset, dest);
+	}
+	catch (vmime::exception &e) {
+		// copy 'word' as text
+		out = m_buffer;
+	}
 
 	return (out);
 }
